Src/Particle/AMReX_Particle.H

```diff
@@ -15,13 +15,14 @@ namespace amrex {
 
 namespace
 {
-    /** Used for 64bit Long particle Ids as in AoS layout */
+    /** Special flags used for 64-bit Long particle Ids */
     namespace LongParticleIds {
         constexpr Long GhostParticleID = 549755813887L; // 2**39-1
```

Comment:
Would then be
```suggestion
        constexpr Long GhostParticleID = 549755813887LL; // 2**39-1
```
or
```suggestion
        constexpr Long GhostParticleID = 549755813887U; // 2**39-1
```
https://en.cppreference.com/w/cpp/language/integer_literal